RadixWallet/Clients/TransactionClient/Models/TransactionModels.swift

```diff
@@ -38,14 +38,14 @@ extension GatewayAPI.TransactionPreviewRequest {
 		let notaryIsSignatory = transactionSigners.notaryIsSignatory
 
 		try self.init(
-			manifest: rawManifest.instructions().asStr(),
+			manifest: rawManifest.instructionsString(),
 			blobsHex: rawManifest.blobs().map(\.hex),
-			startEpochInclusive: .init(header.startEpochInclusive),
-			endEpochExclusive: .init(header.endEpochExclusive),
+			startEpochInclusive: .init(header.startEpochInclusive.rawValue),
```

Comment:
This is constructing a `Gateway.TransactionHeader` which works with literals and low level types, we see a git diff here since I've upgraded the types of the (RET/Sargon)TransactoinHeader to use strong types, e.g. `NetworkID` instead of `UInt8`.